src/Components/Popover/Base/PopoverDialog.tsx

```diff
@@ -146,6 +147,20 @@ export const PopoverDialog = forwardRef<PopoverDialogRef, PopoverDialogProps>(
       handleUpdateStyles()
     })
 
+    // Ensure styles are updated when the
+    // enableDefaultStyles prop changes
+    useEffect(() => {
+      handleUpdateStyles()
+    }, [enableDefaultStyles])
+
+    // Ensure dialog element is in focus on mount
+    // in order to enable escape key behavior
+    useEffect(() => {
+      if (dialogRef.current) {
+        dialogRef.current.focus()
```

Comment:
Could this inadvertently pull focus from an element within the dialog, like a button or input field?

Reply:
That's a good question... I'll try building an example of this in storybook